website/src/pages/index.tsx

```diff
@@ -22,59 +22,59 @@ type FeatureItem = {
 
 const FeatureList: FeatureItem[] = [
   {
-    title: "Pub/Sub logging",
+    title: "Pub/sub logging",
     Icon: icons.Robot,
     description:
-      "MCAP is ideal for storing multiple channels of timestamped log data, such as pub/sub messages or multimodal sensor data.",
+      "Store multiple channels of timestamped log data, including pub/sub messages or multimodal sensor data.",
   },
 
   {
-    title: "Serialization agnostic",
+    title: "Serialization-agnostic container format",
```

Comment:
👎 on this change, the title is super long now, and it moves part of the explanation into the title. I would stick with the simpler title.